### PR TITLE
Use csvReporter as well when running BiDi tests locally

### PR DESCRIPTION
**Pull request by whimboo**
@yury-s mind a review? Thanks!

### CODE REVIEW

**Comment by whimboo** [comment]
@microsoft-github-policy-service agree company="Mozilla"

File: .gitignore [hunk 1]
```diff
@@ -32,6 +32,7 @@ test-results
 .env
 /tests/installation/output/
 /tests/installation/.registry.json
+/tests/**/*.ts-snapshots/
```
Comment (yury-s) [inline .gitignore:35]: Is this change intentional? I don't see any directories with such names in my checkout.

**Review by yury-s — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (whimboo) [reply .gitignore:35]: I can split this out if you want. I have a lot of these after running the screenshot tests as example:

```
	tests/page/page-screenshot.spec.ts-snapshots/page-screenshot-should-capture-css-transform-1-bidi-firefox-nightly.png
	tests/page/page-screenshot.spec.ts-snapshots/remove-should-work-bidi-firefox-nightly.png
```

Comment (yury-s) [reply .gitignore:35]: Oh, we cannot ignore. These are our expectations for each browser. We probably didn't commit expectations for bidi tests as they were failing, but I see at least some of them are there, e.g. `should-capture-css-box-shadow-1-bidi-chromium.png`. Instead of ignoring the directory, we should add missing bidi expectations there. In any case, let's keep it outside of this PR.

Comment (whimboo) [reply .gitignore]: Ok, I removed that particular commit.
